### PR TITLE
Updated hf2_facebook bart large cnn_sshleifer distilbart cnn

### PR DESCRIPTION
**Pull request by Suvratpalt**
**facebook bart large cnn**
Impot Link:
https://ml.azure.com/experiments/id/93e94fef-85a7-48f9-8be3-a0e582091976/runs/coral_quill_46tjx3f952?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Finetune Link:
https://ml.azure.com/runs/mango_comb_stdc1cbr0l?wsid=%2Fsubscriptions%2F72c03bf3-4e69-41af-9532-dfcdc3eefef4%2FresourceGroups%2Fshared-finetuning-rg%2Fproviders%2FMicrosoft.MachineLearningServices%2Fworkspaces%2Fsuvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

https://ml.azure.com/runs/yellow_car_nwzgx6ynsm?wsid=%2Fsubscriptions%2F72c03bf3-4e69-41af-9532-dfcdc3eefef4%2FresourceGroups%2Fshared-finetuning-rg%2Fproviders%2FMicrosoft.MachineLearningServices%2Fworkspaces%2Fsuvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Batch Deploy:
https://ml.azure.com/runs/batchjob-bd59f9f8-8586-4b67-ae72-00a1be72163d?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Blob Link:
https://ml.azure.com/runs/busy_helmet_3m4lywxdxz?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Endpoint Link:
https://ml.azure.com/endpoints/realtime/facebook-bart-large-cnn-1/detail?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

facebook-bart-large-cnn-1703075

conda Link: https://ml.azure.com/model/facebook-bart-large-cnn:8/artifacts?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

**conda.yaml**

```
channels:
channels:
- conda-forge
dependencies:
- python=3.8.18
- pip<=23.3.1
- pip:
  - mlflow==2.8.1
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.8.1
  - azureml-core==1.54.0.post1
  - azureml-mlflow==1.54.0.post1
  - azureml-metrics[all]==0.0.38.post1
  - scikit-learn==1.1.3
  - cryptography==41.0.7
  - python-dateutil==2.8.2
  - datasets==2.15.0
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.24.0
  - sentencepiece==0.1.99
  - transformers==4.36.2
  - torch==1.13.1
  - accelerate==0.25.0
  - Pillow==10.0.1
  - azureml-evaluate-mlflow==0.0.38
- pycocotools=2.0.4
name: mlflow-env

```

**MLmodel**

```
flavors:
  hftransformersv2:
    code: null
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForSeq2SeqLM
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: facebook/bart-large-cnn
    model_data: data
    pytorch_version: 1.13.1
    task_type: summarization
    transformers_version: 4.36.2
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.18
metadata:
  base_model_name: facebook/bart-large-cnn
  base_model_task: summarization
  is_acft_model: true
  is_finetuned_model: false
mlflow_version: 2.8.1
model_uuid: 2712c73247654237a6c348f3f953631b
utc_time_created: '2023-12-20 12:01:34.122189'

```

**sshleifer-distilbart-cnn**
Import Link:
https://ml.azure.com/runs/bold_zebra_4mfyqyyv1b?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Finetune Link:
https://ml.azure.com/runs/polite_tree_m49bm03psm?wsid=%2Fsubscriptions%2F72c03bf3-4e69-41af-9532-dfcdc3eefef4%2FresourceGroups%2Fshared-finetuning-rg%2Fproviders%2FMicrosoft.MachineLearningServices%2Fworkspaces%2Fv-suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

https://ml.azure.com/runs/blue_animal_50721fr6ml?wsid=%2Fsubscriptions%2F72c03bf3-4e69-41af-9532-dfcdc3eefef4%2FresourceGroups%2Fshared-finetuning-rg%2Fproviders%2FMicrosoft.MachineLearningServices%2Fworkspaces%2Fv-suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Batch Deploy:
https://ml.azure.com/runs/batchjob-273d6d72-b1fd-4e9e-b2a7-9a0f2118c404?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Blob Link:
https://ml.azure.com/runs/frosty_pin_4n6vg8xl4v?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Endpoint Link:
https://ml.azure.com/endpoints/realtime/sshleifer-distilbart-cnn/detail?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

sshleifer-distilbart-cnn-12-6-1

Conda file Link: https://ml.azure.com/model/sshleifer-distilbart-cnn-12-6:4/artifacts?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-suvrat&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

**conda.yaml**

```
channels:
- conda-forge
dependencies:
- python=3.8.18
- pip<=23.3.1
- pip:
  - mlflow==2.8.1
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.8.1
  - azureml-core==1.54.0.post1
  - azureml-mlflow==1.54.0.post1
  - azureml-metrics[all]==0.0.38.post1
  - scikit-learn==1.1.3
  - cryptography==41.0.7
  - python-dateutil==2.8.2
  - datasets==2.15.0
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.24.0
  - sentencepiece==0.1.99
  - transformers==4.36.2
  - torch==1.13.1
  - accelerate==0.25.0
  - Pillow==10.0.1
  - azureml-evaluate-mlflow==0.0.38
- pycocotools=2.0.4
name: mlflow-env

```

**MLmodel**

```
flavors:
  hftransformersv2:
    code: null
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForSeq2SeqLM
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: sshleifer/distilbart-cnn-12-6
    model_data: data
    pytorch_version: 1.13.1
    task_type: summarization
    transformers_version: 4.36.2
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.18
metadata:
  base_model_name: sshleifer/distilbart-cnn-12-6
  base_model_task: summarization
  is_acft_model: true
  is_finetuned_model: false
mlflow_version: 2.8.1
model_uuid: d5b585f518fd49528166bcef880a82a6
utc_time_created: '2024-01-02 06:09:05.268143'

```

### CODE REVIEW

File: assets/models/system/facebook-bart-large-cnn/model.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/facebook-bart-large-cnn/3d22493/mlflow_model_folder
+  container_path: huggingface/facebook-bart-large-cnn/1703165534/mlflow_model_folder
   storage_name: automlcesdkdataresources
   type: azureblob
```
Comment (BhavanaTumma) [inline assets/models/system/facebook-bart-large-cnn/model.yaml:5]: This is still pointing to the version with mlflow version 2.3.1, please save the right model

File: assets/models/system/sshleifer-distilbart-cnn-12-6/model.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/sshleifer-distilbart-cnn-12-6/a4f8f3e/mlflow_model_folder
+  container_path: huggingface/sshleifer-distilbart-cnn-12-6/1703564895/mlflow_model_folder
   storage_name: automlcesdkdataresources
```
Comment (BhavanaTumma) [inline assets/models/system/sshleifer-distilbart-cnn-12-6/model.yaml:4]: These models have Whisper packages, please refer to other models and remove them if not required.

File: assets/models/system/facebook-bart-large-cnn/model.yaml (hunk 1 above)
Comment (Suvratpalt) [reply assets/models/system/facebook-bart-large-cnn/model.yaml:5]: Updated mlflow_version: 2.8.1 with change Blob Link

**Review by Suvratpalt — COMMENTED** [review]
Updated mlflow_version: 2.8.1 with changes Blob Link

File: assets/models/system/sshleifer-distilbart-cnn-12-6/model.yaml (hunk 2 above)
Comment (Suvratpalt) [reply assets/models/system/sshleifer-distilbart-cnn-12-6/model.yaml:4]: Hi Bhavana, now Updated Please check Thanks.

**Comment by KonakanchiSwathi** [comment]
closing as its  not required

**Pull request closed without merging**